app/domain/people/data_quality_checker.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+class People::DataQualityChecker
```

Comment:
Would be cleaner if we defined the attributes to check like this

```
ATTRIBUTES_TO_CHECK = {
    "street" => { severity: "error", key: "empty", invalid: @person.street.blank, perform_check:  @person.sac_membership_invoice?}
  }
```
  
  and then call check like this
  
```
  ATTRIBUTES_TO_CHECK.each do |field, data|
    check(data[:severity], field, data[:message], invalid: data[:invalid]) if data[:perform_check]
  end
  
```

Reply:
`ATTRIBUTES_TO_CHECK` cannot be a hash because `:birthday` must be checked twice

Reply:
Didn't test my solution, was only a quick suggestion, but your solution now is good :+1: